server/emailsWithAttached.js

```diff
@@ -0,0 +1,23 @@
+const AWS = require('aws-sdk');
+const nodeMailer = require('nodemailer');
+
+async function sendMailWithAttched(mailingData) {
+    const sourceEmail = 'ourbrother@connect2care.ourbrothers.co.il';
+
+    let transporter = nodeMailer.createTransport({
+        SES: new AWS.SES({ region: 'eu-west-1', apiVersion: '2010-12-01'})
+    });
+    console.log('Trying to send email', mailingData.to, mailingData.subject)
+    let mail = await transporter.sendMail({
+        from: sourceEmail,
+        to: mailingData.to,
+        subject: mailingData.subject,
+        html: mailingData.html,
+        attachments: mailingData.attachedFiles
+
+    });
+    console.log("Message sent: %s", mail.messageId);
+    return mail;
+}
+
+module.exports = sendMailWithAttched;
```

Comment:
missing new line at the end

Reply:
What do you mean "missing new line at the end" ?

Reply:
It is customary to add a new line at the end of each file (there are actual reasons for that but non of them concerns you right now). It is a good practice nevertheless...